src/force/nep3.cu

```diff
@@ -794,15 +855,22 @@ static __global__ void find_force_radial(
       s_fx += f12[0] - f21[0];
       s_fy += f12[1] - f21[1];
       s_fz += f12[2] - f21[2];
-      s_sxx += r12[0] * f21[0];
+      if (is_dipole) {
+        double r12_square = r12[0] * r12[0] + r12[1] * r12[1] + r12[2] * r12[2];
+        s_sxx += r12_square * f21[0];
```

Comment:
should be `s_sxx += r12_square * f21[0];` etc, and all other similar places for dipole